chatbot/index.js

```diff
@@ -80,9 +80,14 @@ function localize([inputFlow, translations, lang, outputName, outputDir]) {
 }
 
 
-function move_quick_replies([input_file, select_phrases, outputName, outputDir,add_selectors = false]) {
+function move_quick_replies([input_file, select_phrases, outputName, outputDir, add_selectors = "no", special_words = false]) {
```

Comment:
It's not possible for more than 1 argument to be optional. If I don't specify a value for 'add selectors' (because the default is "no"), but specify a path for 'special words', then `special_words` will be `undefined` and `add_selectors` will be the path to the special words file.

I suggest making add_selectors a required argument - it must be either yes/no (or true/false, whatever you decide to go with), but it cannot be left out. The only truly optional argument is the last one in the list.

In future we should consider using a CLI arguments parsing library like 'yargs', if the arguments continue to get more sophisticated.

Reply:
Understood, will make it a required argument "yes/no"